### DISCUSSION TITLE
Migrated 6.2 to 6.3 API error

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15693] [General]
Hi

Just quick question. Migrated to 6.3 and API connection designed for previous version returning error as below.
No time right now to debug error but curious if that is something known/expected after migration.

2022-02-06 10:21:55.4426 [error] - Web service - Applications: Unauthorized
Api\Core\Exception: Web service - Applications: Unauthorized in /var/www/html/api/webservice/Core/Auth/Basic.php:24
Stack trace:
#0 /var/www/html/api/webservice/Core/Auth.php(39): Api\Core\Auth\Basic->authenticate('YetiForceApi')
#1 /var/www/html/api/webservice/Controller.php(87): Api\Core\Auth::init(Object(Api\Controller))
#2 /var/www/html/webservice.php(23): Api\Controller->preProcess()
#3 {main}
 - 0 webservice.php (29)  >>  App\Log->error('Web service - Applications: Unauthorized
Api\\Core\\Exception: Web service - Applications: Unauthorized in api/webservice/Core/Auth/Basic.php:24
Stack trace:
#0 api/webservice/Core/Auth.php(39): Api\\Core\\Auth\\Basic->authenticate(\'YetiForceApi\')
#1 api/webservice/Controller.php(87): Api\\Core\\Auth::init(Object(Api\\Controller))
#2 webservice.php(23): Api\\Controller->preProcess()
#3 {main}')
2022-02-06 10:21:55.4396 [info][application] - 
$_GET = [
    '_container' => 'WebserviceStandard'
    'action' => ''

**Comment by s2b-git** [comment]
Basically it stopped showing server as per screenshot below:

![image](https://user-images.githubusercontent.com/84110093/152670690-6c323edb-a571-43a5-aac8-6a0fbbac3279.png)

**Reply by waran70** [reply]
Breaking API between single update numbers is not funny at all.

**Reply by mariuszkrzaczkowski** [reply]
who is joking?
development and expansion sometimes require major changes
without this change it was not possible to fix api compatibility bug on other platforms

**Reply by rskrzypczak** [reply]
https://doc.yetiforce.com/api/

> rewrite
CRM_URL/webservice/**WebserviceStandard**/Users/Login
without rewrite
CRM_URL/webservice.php?_container=**WebserviceStandard**&module=Users&action=Login

**Reply by s2b-git** [reply]
This helped. Thank you
